tests/test-cli.rs

```diff
@@ -135,6 +135,19 @@ fn test_markdown_help() {
     assert!(output.status.success(), "{}", output.status);
 }
 
+trait CommandExt {
+    fn set_cache(&mut self) -> &mut Self;
+}
+
+impl CommandExt for Command {
+    fn set_cache(&mut self) -> &mut Self {
+        self.arg("--cache-dir")
+            .arg("../cache")
+            .arg("--current-time")
+            .arg("2023-01-01 12:00:00Z")
+    }
+}
+
```

Comment:
I wonder if we should make this also pass in `--manifest-path` and call it something like `test_project_args()`?

It looks like we have some `diff`-style calls which are probably running against the wrong repo right now because they don't specify the manifest-path, which is interesting, but should probably be fixed. (The `diff` command previously didn't care about the repo being acted upon, and these command lines are probably from when that was the case)

Reply:
I was curious whether those inconsistencies were intentional or not. It'd be good to make them all consistent.